db/queries/assessment_records/_helpers.py

```diff
@@ -21,27 +22,83 @@ def derive_application_values(application_json):
         )
         asset_type = "Not asset type specified."
     try:
-        funding_one = (
-            jsonpath_rw_ext.parse(
-                '$.forms[*].questions[*].fields[?(@.key =="JzWvhj")]'
+        if "COF-R2" in short_ref:  # cof r2
```

Comment:
There's quite a bit of duplicated code here - could you have a small function that gets the value of an answer and keep calling that? Easier to mock and test then too. eg.
```
def get_answer_value(answer_key):
                    return jsonpath_rw_ext.parse(
                        '$.forms[*].questions[*].fields[?(@.key == answer_key)]'
                    )
                    .find(application_json)[0]
                    .value["answer"]
```

Reply:
Done now